projects/plugins/automattic-for-agencies-client/src/js/components/connection-card/index.jsx

```diff
@@ -1,61 +1,333 @@
+					{ __( 'See your site in the Sites dashboard', 'automattic-for-agencies-client' ) }
+				</li>
+				<li>
+					<CheckIcon />
 					{ __(
-						'Add this site to Automattic for Agencies by connecting now',
+						'View any security and performance issues across all of your sites',
 						'automattic-for-agencies-client'
 					) }
-				</h1>
-				<ul className={ styles[ 'check-list' ] }>
+				</li>
+				<li>
+					<CheckIcon />
+					{ __(
+						'Update plugins across all sites in a couple of clicks',
+						'automattic-for-agencies-client'
+					) }
+				</li>
+				<li>
+					<CheckIcon />
+					{ __( 'Receive instant downtime alerts', 'automattic-for-agencies-client' ) }
+				</li>
+				<li>
+					<CheckIcon />
+					{ __( 'And more', 'automattic-for-agencies-client' ) }
+				</li>
+			</ul>
+			<div className={ styles[ 'site-connection' ] }>
+				<div className={ styles[ 'terms-of-service' ] }>
+					{ createInterpolateElement(
+						__(
+							'By clicking the <strong>connect this site</strong> button, you agree to our <tosLink>Terms of Service</tosLink> and to <shareDetailsLink>share details</shareDetailsLink> with WordPress.com.',
+							'automattic-for-agencies-client'
+						),
+						{
+							strong: <strong />,
+							tosLink: (
+								<a
+									className={ styles[ 'terms-of-service__link' ] }
+									href={ getRedirectUrl( 'wpcom-tos' ) }
+									rel="noopener noreferrer"
+									target="_blank"
+								/>
+							),
+							shareDetailsLink: (
+								<button
+									className="components-button is-link"
+									onClick={ onShowSharingDetailsClick }
+								/>
+							),
+						}
+					) }
+				</div>
+				<div className={ styles[ 'connect-button-wrapper' ] }>
+					<ConnectButton
+						connectLabel={ __( 'Connect this site', 'automattic-for-agencies-client' ) }
+						apiRoot={ apiRoot }
+						apiNonce={ apiNonce }
+						registrationNonce={ registrationNonce }
+						from="automattic-for-agencies-client"
+						redirectUri="admin.php?page=automattic-for-agencies-client"
+					/>
+				</div>
+			</div>
+		</>
+	);
+}
+
+/**
+ * Detail Sharing Content
+ * The copy for the "share details" information linked from the TOS.
+ *
+ * @param {object}   props                             - Component props
+ * @param {Function} props.onCloseSharingDetailsClick  - Callback to close the sharing details.
+ *
+ * @returns {React.Component} The `DetailSharingContent` component.
+ */
+function DetailSharingContent( { onCloseSharingDetailsClick } ) {
+	return (
+		<>
+			<div>
+				<button
+					className="components-button components-button--back is-link"
+					onClick={ onCloseSharingDetailsClick }
+				>
+					<LeftArrow />
+					{ __( 'Go back to site connection', 'automattic-for-agencies-client' ) }
+				</button>
+			</div>
+			<div className={ styles.card__prose }>
+				<h2>What data is synced from your site</h2>
+				<p>
+					Your privacy matters to us. We sync only the necessary data required to provide our
```

Comment:
This would need to be translatable.

Reply:
Whoops, thanks for catching these untranslated strings. Fixed via https://github.com/Automattic/jetpack/pull/36729/commits/4802b612a786c9d82b46256d18533aaf9210b710 👍